server/lib/friends.ts

```diff
@@ -242,6 +242,24 @@ function fetchRemotePreview (pod: PodInstance, video: VideoInstance) {
   return request.get(REMOTE_SCHEME.HTTP + '://' + host + path)
 }
 
+function removeFriend (pod: PodInstance) {
+  const requestParams = {
+    method: 'POST' as 'POST',
+    path: '/api/' + API_VERSION + '/remote/pods/remove',
+    sign: true,
```

Comment:
You don't need the `sign` parameter (it was removed recently, now all requests are automatically signed).